<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>317</height>
   </rect>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>26</y>
     <width>499</width>
     <height>268</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <widget class="QTreeWidget" name="treeCluster" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="contextMenuPolicy" >
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <column>
       <property name="text" >
        <string>Clusters</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabsMain" >
      <property name="minimumSize" >
       <size>
        <width>325</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip" >
       <string/>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="MTasks" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>321</width>
         <height>222</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Management Tasks</string>
       </attribute>
       <attribute name="toolTip" >
        <string>Cluster Management Tasks</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QGroupBox" name="gbClearKey" >
          <property name="title" >
           <string>Delete Key(s)</string>
          </property>
          <property name="flat" >
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2" >
           <item row="0" column="0" >
            <widget class="QLabel" name="lblCacheKeys" >
             <property name="text" >
              <string>Cache Key(s)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLineEdit" name="txtCacheKeys" >
             <property name="toolTip" >
              <string>Seperate Keys with ';'</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QPushButton" name="btnCacheKeys" >
             <property name="text" >
              <string>Delete Key(s)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbTasks" >
          <property name="title" >
           <string>Server Admin Tasks</string>
          </property>
          <layout class="QGridLayout" name="gridLayout" >
           <item row="0" column="0" >
            <widget class="QPushButton" name="btnFlushCache" >
             <property name="toolTip" >
              <string>Flush Keys from All Servers in Cluster</string>
             </property>
             <property name="text" >
              <string>Flush Cache Keys</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SKInfo" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>321</width>
         <height>222</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Slab &amp;&amp; Key Info</string>
       </attribute>
       <attribute name="toolTip" >
        <string>Slabs, Keys, &amp; Values</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Stats" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>321</width>
         <height>222</height>
        </rect>
       </property>
       <attribute name="title" >
        <string>Statistics</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionSave" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menu_Servers" >
    <property name="title" >
     <string>&amp;Servers</string>
    </property>
    <addaction name="actionAddServer" />
    <addaction name="actionAddCluster" />
    <addaction name="separator" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menu_Servers" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>294</y>
     <width>499</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionAddServer" >
   <property name="text" >
    <string>&amp;Add Server</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="text" >
    <string>Sa&amp;ve</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAddCluster" >
   <property name="text" >
    <string>Add &amp;Cluster</string>
   </property>
  </action>
  <action name="actionTest_Sub_Item" >
   <property name="text" >
    <string>Test Sub Item</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>295</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
